dbschema/alias.esdl

```diff
@@ -0,0 +1,10 @@
+module default {
+  type Alias {
```

Comment:
I'm not convinced this should be in the default module., I also don't think we have an existing module that fits this. Thoughts?

Reply:
How will this be used within other areas of the schema?  Can you present an example?

Reply:
Well I do think it could be good to keep this as a standalone thing.
I didn't add an `aliases` backlink anywhere in schema.

It could be used to provide a legacy ID for DOMO, but that would take some more work to pass through the API and in every query.
At one point in my dev I thought that this alias could be integrated into the `readMany` queries. I wanted it to be transparent, so that seemed like a not bad way to do it.
But it didn't prove out well, so I iterated to this approach seen in PR.
But the query was something like
```edgeql
with ids := {"carson", "9b84a0ba-bbba-11ee-9de2-c7870f73b88d", "root"},
  items := (
    (select User filter <str>.id in ids)
    union
    (select Alias filter .name in ids).target[is User]
  )
select items {
  id,
  legacyId := (select .<target[is Alias] filter .name in ids limit 1).name ?? <str>.id,
  ...
}
```

----


My point with this comment though was to question whether this was really a top level feature of CORD or rather something of lesser significance and therefore doesn't belong in the `default` module.
Rather `Helper::Alias` or `Util::Alias`. But both of those suck IMO lol.

Reply:
Yeah, that's why I asked.  I was trying to see more how it would be used so I could better think about where it should live.  I was wondering about it being a Mixin, but after seeing this usage that doesn't make sense.  It really is some kind of "util" function.  Whatever we do, please don't use `Helper` for anything.  lol.  It's a pet peeve of mine...helper of what?  I'd rather go `Util::` for sure.  

However, I could almost see it just staying in default.  I'd even consider just putting it in `common.esdl` maybe?

Reply:
Ok let's just keep it in default for now. Meh I think it's distinct enough to have it's own file. I don't want common to blow up like a util folder.

Reply:
I'm fine with either.  It'll all become more clear as we progress anyway.